api.planx.uk/modules/flows/routes.ts

```diff
@@ -71,4 +74,10 @@ router.get(
   downloadFlowSchemaController,
 );
 
+router.get(
+  "/flows/:flowId/flatten-data",
+  validate(flattenFlowData),
+  flattenFlowDataController,
+);
```

Comment:
Right - this endpoint is public because `flows.data` & `published_flows.data` are publicly select-able in our database. Nothing to do with submission functionality here, that's handled by the frontend views & routing.

I've already been finding this endpoint to be a handy utility for copying flow data, generating mock flow data for tests etc.